### PR TITLE
Conversion

### PR DESCRIPTION
**Pull request by yusufidimaina9989**

### CODE REVIEW

**Review by xhliu — CHANGES_REQUESTED** [review]
👍 awesome, code looks good.

Can you please do the following?

- move contracts to src/contracts
- add test for each contract under tests, both local and testnet.

**Comment by yusufidimaina9989** [comment]
open a new or

File: numbergs.testnet.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Numbergs } from '../../src/contracts/numbergs'
```
Comment (xhliu) [inline numbergs.testnet.ts:1]: Move to test/ folder

File: BlockTimeBet.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import {assert, prop, method, SmartContract,Sha256, PubKey, Sig, FixedArray, checkSig} from 'scrypt-ts'
```
Comment (xhliu) [inline BlockTimeBet.ts:1]: Put all these contracts under `sr/contracts/`

Comment (xhliu) [inline BlockTimeBet.ts:1]: Put all these contracts under `sr/contracts/`

**Review by xhliu — APPROVED** [review]
👍

**Review by xhliu — APPROVED** [review]
👍 well done

**Pull request merged**